packages/devextreme/js/common/ai-integration.d.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface Prompt {
+  /**
+   * @docid
+   * @public
+   */
+  system?: string;
+  /**
+   * @docid
+   * @public
+   */
+  user?: string;
+}
+
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface RequestParams {
+  /**
+   * @docid
+   * @public
+   */
+  prompt: Prompt;
+  /**
+   * @docid
+   * @public
+   */
+  onChunk: (chunk: string) => void;
+}
+
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface ResponseParams {
+  /**
+   * @docid
+   * @public
+   */
+  promise: Promise<string>;
+  /**
+   * @docid
+   * @public
+   */
+  abort: () => void;
+}
+
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface TranslateCommandParams {
```

Comment:
If it's not a problem with generation or usage scenaros, we still should follow this rule:

> All public types are `type`s (type aliases, not `interface`s) to disallow [interface merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces) on end project.
> It's also because type aliases are better suited for creating a public facade. In the future if we need to change a public type from `interface` to `type` (e.g. to make it a discriminated union) we will add potential BC. On the other hand, converting from `type` to `interface` is expected to be BC-less

```suggestion
export type TranslateCommandParams = {
```

Reply:
Same for other public `interface`'s: Prompt, RequestParams, ResponseParams, RequestCallbacks, interface, AIProvider

Reply:
I have no objection